tests/utils.go

```diff
@@ -789,17 +790,28 @@ func AdjustKubeVirtResource() {
 			PanicOnError(err)
 		}
 	}
+
+	return adjustedKV
 }
 
-func RestoreKubeVirtResource() {
-	if originalKV != nil {
+func RestoreKubeVirtResource(kv *v1.KubeVirt) {
```

Comment:
I kind think that this change should rename the function as well, since now it became not a restore, but an overwrite - or something like that.